### ISSUE TITLE
Add a note to the README with flight heritage of this framework

### ISSUE THREAD

**Issue by DeflateAwning** [olmanqj/sfsf#2]
We are considering using this framework on an upcoming cubesat mission, but want to fully understand how far along its development is before we commit to it.

Can you please add a note to the README which indicates any cubesat missions this framework has been used on? Ideally, links to project pages would also be helpful!

**Comment by olmanqj** [comment]
Thanks so much for the interest! The framework is not being developed nor maintained anymore. I developed the code for a CubeSat project which started in 2018, however the project was never finished. So till the date, the framework has not yet been used in orbit. 

That being said, if you end up choosing this framework for your project, I could allocate some work hours to continue the development/maintenance, hand in hand with your project/team. I believe the framework still have a lot of potential, specially for student projects and teaching.

**Comment by DeflateAwning** [comment]
Thanks a lot for the offer! We're quite interested, and would love to be able to help continuing this framework.

However, we want to make sure we understand the state of the software before we invest significant resources in a potentially unstable framework/foundation for the project. How extensively has the existing framework been tested? Do you know if it's been used in more than one mission/codebase? Do you know if there's a repo which shows example usage of this framework? Thanks!

**Comment by olmanqj** [comment]
> How extensively has the existing framework been tested? 

Being honest, the verification status is poor.  I tested it manually, by implementing the Flight Software of the CubeSats and trying out all commands, scenarios, etc. Unfortunately, I didn't write any unittest or integration test, but I'm interested in doing it.

> Do you know if it's been used in more than one mission/codebase?
 
As far as I know, No. Just in the CubeSat I was working on.

> Do you know if there's a repo which shows example usage of this framework? 

The [examples](https://github.com/olmanqj/sfsf/tree/master/examples) directory shows an example of a simple ground terminal which will receive telemetry and can send commands from/to a "Space segment", which is implemented as a Linux process.

**Comment by DeflateAwning** [comment]
Has this code ever been run on a microcontroller/custom on-board computer for a CubeSat? Or was it only used in the early development stages? Can you suggest how much work it would be to port it for STM32?

**Comment by olmanqj** [comment]
> Has this code ever been run on a microcontroller/custom on-board computer for a CubeSat? 

Yes, in a custom OBC with an AVR32. The drivers for CSP were not written by me and that was kept private. 

> Can you suggest how much work it would be to port it for STM32?

Porting the framework is basically porting [CPS (the network layer)](https://github.com/libcsp/libcsp). 
For porting CSP there are two aspects: the OS and the communication interface.
- CSP can use several interfaces for reading sending packages, the recommendation would be to keep it simple and just use USART, that means to write a driver like any of the already existent, [see these examples] (https://github.com/libcsp/libcsp/tree/develop/src/drivers/usart). I consider this is not much work, ~200 lines of code.
-  Regarding the OS, CSP is implemented for running in FreeRTOS, Linux and Windows. Although running it in other OSes is possible, the recommendation is using any of the two first ones. If you stick to that, no porting effort is needed.
